mne/evoked.py

```diff
@@ -115,6 +117,37 @@ def __init__(self, fname, condition=None, baseline=None, proj=True,
             self.apply_proj()
         self.data = rescale(self.data, self.times, baseline, copy=False)
 
+    @verbose
+    def apply_baseline(self, baseline, verbose=None):
+        """Baseline correct evoked data
+
+        Parameters
+        ----------
+        baseline : tuple of length 2
```

Comment:
PR == many commits

so push your commits in the branch to append the commits. We'll merge when
it's complete